web-platform/website/src/pages/login.tsx

```diff
@@ -20,16 +21,20 @@ const strings = Strings.pages.login;
 
 const LoginPage: NextPage = () => {
   const { isLoaded, user } = useAuthentication();
-
   const router = useRouter();
-
   const { isFlagActive } = useFlagService();
+  const { installedStudyIds } = useStudies();
 
   if (!isLoaded || !router.isReady) {
     return null;
   }
 
-  if (user) {
+  if (isFlagActive(Flags.onboardingV2)) {
+    if (user && installedStudyIds.length > 0) {
+      router.replace("/");
+      return null;
+    }
```

Comment:
This is already covered by original code unconditionally. So if user is present, we should always route to "/". Hence we can revert this file completely?

Reply:
Where else? The only where I see this is here and it only affects the title of the card not the content:

`function LoginCardTitle() {
  const { loginState } = useLoginDataContext();
  const { installedStudyIds } = useStudies();

  if (
    loginState === LoginState.Login ||
    loginState === LoginState.ResetPassword ||
    loginState === LoginState.GetExtension
  ) {
    return null;
  }

  const { title, subtitle } = installedStudyIds.length
    ? strings.titles.extensionFirst
    : strings.titles.accountFirst;

  return (
    <>
      <Row className="mb-4">
        <Col className="d-flex justify-content-center">
          <Highlighter className={`w-100 text-left ${styles.v2Highlighter}`}>
            <h1 className={Fonts.Headline}>{title}</h1>
          </Highlighter>
        </Col>
      </Row>
      <Row className="mb-4">
        <Col>
          <h5>{subtitle}</h5>
        </Col>
      </Row>
    </>
  );
}`

Reply:
For the `GetExtensionView `page to appear I need to account for the user because this page shows up _post account creation_ and I must also detect whether the extension is installed or not...

Reply:
I've decided to make the `GetExtensionView` its own page since it requires an authenticated user and is actually separate from the login flow. Just used your new split layout for the styles.

Reply:
I think that is a good decision since we should not mix logged out and logged in user states. Thanks for doing this!

Reply:
For sure!